<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterSettingsWidget</class>
 <widget class="QWidget" name="PrinterSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>530</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <widget class="QTabWidget" name="Config">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>96</y>
     <width>340</width>
     <height>390</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>340</width>
     <height>390</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Connection_Config">
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="cb_ConnectionPort">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="ck_ResetOnConnect">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lb_Protocol">
         <property name="text">
          <string>Firmware:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lb_ResetC">
         <property name="text">
          <string>Reset on Connect:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_Port">
         <property name="text">
          <string>Port of Connection:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_ConnectionType">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="locale">
          <locale language="Portuguese" country="Brazil"/>
         </property>
         <item>
          <property name="text">
           <string>USB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Serial</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lb_log">
         <property name="text">
          <string>Printer Log:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cb_TransmitionRate">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cb_Firmware">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Repetier</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_Transmission">
         <property name="text">
          <string>Transmission Rate:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="ck_PrintLog">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cb_CacheSize">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>127</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lb_Cache">
         <property name="text">
          <string>Cache size:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_Connection">
         <property name="text">
          <string>Connection:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Print_Config">
    <attribute name="title">
     <string>Printer Specs</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>15</y>
       <width>315</width>
       <height>275</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_RateMoviment">
        <property name="text">
         <string>Moviment Rate(mm/min):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="tb_RateMoviment">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_Feed_Z">
        <property name="text">
         <string>Feed Z Axis(mm/min):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="tb_FeedZ">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_ExtruderV">
        <property name="text">
         <string>Extruder Speed(mm/s):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="tb_ExtruderSpeed_mm">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lg_Velocity_R">
        <property name="text">
         <string>Retraction Speed(mm/s):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="tb_SpeedRetraction">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_Extruder_Temper">
        <property name="text">
         <string>Extruder Temperature(ºC):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="tb_ExtruderTemperature">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_TableTemperature">
        <property name="text">
         <string>Bed Temperature(ºC):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="tb_BedTemperature">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>130</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lb_areax">
        <property name="text">
         <string>Print Area - Width(mm)(X):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="tb_AreaPrintX">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lb_areay">
        <property name="text">
         <string>Print Area - Height(mm)(Y):</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="tb_AreaPrintY">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lb_areaz">
        <property name="text">
         <string>Print Area - Vertical(mm)(Z):</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="tb_AreaPrintZ">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Extrusor_Config">
    <attribute name="title">
     <string>Extruder</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>315</width>
       <height>326</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="lb_obs">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align: center;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Check with the manufacturer&lt;br/&gt; or the specifications of your&lt;br/&gt;printer to set Max temperatures on bed &lt;br/&gt;and (s) extruder(s). &lt;br/&gt;As for security, if any error&lt;br/&gt; occurs systemic or printer,&lt;br/&gt;these variables &lt;br/&gt;serve for emergency shutdown.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_vol_sec">
        <property name="text">
         <string>Volume Max per Second(mm³/s):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_extruder_number">
        <property name="text">
         <string>Number of Extruders:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cb_ExtruderQnt">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_tabletmp">
        <property name="text">
         <string>Bed Max Temperature(ºC):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_extrudertmp">
        <property name="text">
         <string>Extruder Max Temperature(ºC):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Icons/Icons/warningMax.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="tb_BedMaxTemp">
        <property name="maximum">
         <number>130</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="tb_ExtruderMaxTemp">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="tb_ExtruderMaxVol">
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>125</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer Settings</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>331</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="lb_printer">
      <property name="text">
       <string>Settings:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cb_PrinterSettings">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="bt_SaveSettings">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
